app/Http/Controllers/Dashboard/IncidentController.php

```diff
@@ -107,19 +111,25 @@ public function showTemplates()
      */
     public function createIncidentAction()
     {
-        $incidentData = Binput::get('incident');
-        $componentStatus = array_pull($incidentData, 'component_status');
-
-        if (array_has($incidentData, 'created_at') && $incidentData['created_at']) {
-            $incidentDate = Date::createFromFormat('d/m/Y H:i', $incidentData['created_at'], Setting::get('app_timezone'))->setTimezone(Config::get('app.timezone'));
-            $incidentData['created_at'] = $incidentDate;
-            $incidentData['updated_at'] = $incidentDate;
-        } else {
-            unset($incidentData['created_at']);
+        if ($createdAt = Binput::get('created_at')) {
```

Comment:
@jbrooksuk I would move this functionality to the handler like we did on the AddMetricPointCommandHandler, what do you think?